src/markdownwidget/widget.ts

```diff
@@ -31,6 +35,11 @@ import {
  */
 const MD_CLASS = 'jp-MarkdownWidget';
 
+/**
+ * The timeout to wait for change activity to have ceased before rendering.
+ */
+const RENDER_TIMEOUT = 350;
```

Comment:
Let's use 1000 for slower typing folks: http://ux.stackexchange.com/questions/38543/amount-of-time-to-determine-a-user-has-stopped-typing